Source/Core/DolphinQt/FIFO/FIFOPlayerWindow.cpp

```diff
@@ -117,10 +117,21 @@ void FIFOPlayerWindow::CreateWidgets()
   auto* playback_group = new QGroupBox(tr("Playback Options"));
   auto* playback_layout = new QGridLayout;
   m_early_memory_updates = new QCheckBox(tr("Early Memory Updates"));
+  m_early_memory_updates->setChecked(Config::Get(Config::MAIN_FIFOPLAYER_EARLY_MEMORY_UPDATES));
+  m_early_memory_updates->setToolTip(
+      tr("If enabled, then all memory updates happen at once before the first frame.\n"
+         "Causes issues with many fifologs, but can be useful for testing.\n"
+         "If unsure, leave this unchecked."));
```

Comment:
If I'm remembering right, we might want

`<dolphin_emphasis>If unsure, leave this unchecked</dolphin_emphasis>`

Reply:
That only works with BalloonTip, but it looks like I can use that here (by using `ToolTipCheckBox` instead of `QCheckBox`).